inc/mailcollector.class.php

```diff
@@ -1184,37 +1222,27 @@ function listEncodings() {
    function decodeMimeString($mimeStr, $inputCharset = 'utf-8', $targetCharset = 'utf-8',
```

Comment:
If needed, may usage may be replaced by `Zend\Mail\Header\HeaderWrap::mimeDecodeValue()` (see https://github.com/zendframework/zend-mail/blob/master/src/Header/HeaderWrap.php#L102)

Fun part is that it seems that when header is on multiple lines, `imap` ext may be usefull (see https://github.com/zendframework/zend-mail/blob/master/src/Header/HeaderWrap.php#L113).

Reply:
Since the subject and the name from contentType header are already handled from Zend Mail, this is no longer needed at all :)